bin/validate/atomic-red-team.schema.json

```diff
@@ -0,0 +1,192 @@
+                        "type": "string",
+                        "enum": [
+                          "Path",
+                          "Url",
+                          "String"
+                        ]
+                      },
+                      "default": {
+                        "type": ["string", "null"]
+                      }
+                    }
+                  },
+                  {
+                    "type": "object",
+                    "properties": {
+                      "description": {
+                        "type": "string"
+                      },
+                      "type": {
+                        "type": "string",
+                        "enum": [
+                          "Integer",
+                          "Float"
+                        ]
+                      },
+                      "default": {
+                        "type": ["number","null"]
+                      }
+                    },
+                    "required": [
+                      "description",
+                      "type",
+                      "default"
+                    ]
+                  }
+                ]
+              }
+            }
+          },
+          "dependency_executor_name": {
+            "type": "string",
+            "enum": [
+              "command_prompt",
+              "powershell",
+              "sh",
+              "bash",
+              "manual"
+            ]
+          },
+          "dependencies": {
+            "type": "array",
+            "unique": true,
+            "items": {
+              "type": "object",
+              "properties": {
+                "description": {
+                  "type": "string"
+                },
+                "prereq_command": {
+                  "type": "string"
+                },
+                "get_prereq_command": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "description",
+                "prereq_command",
+                "get_prereq_command"
+              ]
+            }
+          },
+          "executors": {
+            "type": "array",
+            "unique": true,
+            "items": {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "enum": [
+                    "command_prompt",
+                    "powershell",
+                    "sh",
+                    "bash",
+                    "manual"
+                  ]
+                },
+                "elevation_required": {
+                  "type": "boolean"
+                },
+                "command": {
+                  "type": "string"
+                },
+                "cleanup_command": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "name",
+                "elevation_required",
```

Comment:
elevation_required is optional and assumed to be false if absent

Reply:
Thanks for that. I will update to reflect that.

Reply:
Fixed!